elements/lisk-p2p/README.md

```diff
@@ -1,13 +1,106 @@
 # @liskhq/lisk-p2p
 
-@liskhq/lisk-p2p is containing unstructured P2P library for use with Lisk-related software
+@liskhq/lisk-p2p is containing unstructured P2P library for creating and running unstructured P2P networks. The aim of this library is to make it easier for developers to create P2P projects without thinking about security, discovery, selection and eviction mechanisms and concentrate more on developing use cases and P2P projects on top of it. A developer can leverage underlying robust security, discovery and selections mechanisms and use the easy to use interface to use the library. This library implements features proposed in [LIP: 0004: Introduce robust peer selection and banning mechanism](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0004.md).
 
 ## Installation
 
 ```sh
 $ npm install --save @liskhq/lisk-p2p
 ```
 
+To run a P2P node.
+
+```typescript
+const { P2P } = require('@liskhq/lisk-p2p');
+
+const p2p = new P2P({
+	nodeInfo: {
+		wsPort: 5001,
+		nethash: '123xyz', // network identifier
```

Comment:
maybe use the Devnet nethash? so that by default when someone develops they use Devnet by default and the correct value is more appropriate.?